ui/apps/platform/src/services/ComplianceResultsStatsService.ts

```diff
@@ -28,24 +26,31 @@ export type ListComplianceProfileScanStatsResponse = {
     totalCount: number;
 };
 
+export type ListComplianceClusterProfileStatsResponse = {
```

Comment:
Is this new type separated from the corresponding new service function below?

Reply:
Yes but I'm fine moving it. I've been just putting all type definitions at the top since there has been a lot of overlap, but I can see it making more. sense to at least have the top level response type with its corresponding service function.